recipes/configs/llama2/generation_v2.yaml

```diff
@@ -9,6 +9,10 @@
 # Model arguments
 model:
   _component_: torchtune.models.llama2.llama2_7b
+# You can turn uncomment the following lines to enable quantization for faster inference and potentially lower VRAM
+# quantization_method:
+#   _component_: torchao.quantization.quant_api.int4_weight_only # int4_weight_only is a good balance of speed and memory
```

Comment:
dumb q: so the torchtune.training.quantization API is just for QAT.. or we're not using it anymore?

Reply:
I see you mentioned this in the PR description - if we're going to be using the torchao APIs instead it'd be good to follow up with an issue